### PR TITLE
Remove obsolete Docker instructions

### PR DESCRIPTION
**Pull request by ErikSchierboom**
The script that is being referred to (`./bin/run-docker`) does not exist and is superseded by the `docker-compose` instructions below that I think.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,20 +14,6 @@ The website can be run natively, through Docker, or as part of a wider Docker-co
 
 If you are looking to create a full Exercism setup locally, with tooling such as test-runners and other services, please check out our [development environment instructions](./DEV_ENVIRONMENT.md).
```
Comment (iHiD) [inline README.md]: I suggest rewording this line to say "please check out our [development environment](https://github.com/exercism/development-environment" repo and remove the Docker Compose bit below too.

Comment (ErikSchierboom) [reply README.md]: Done

**Comment by iHiD** [comment]
You've got a stray commit or two on this branch.

**Comment by ErikSchierboom** [comment]
> You've got a stray commit or two on this branch.

Doh. Fixed

**Review by iHiD — APPROVED** [review]
